frontend/src/container/PostsOverviewContainer.vue

```diff
@@ -22,18 +22,12 @@ export default {
     const categorySlug = route.params.slug ? route.params.slug.toString() : null;
 
     const store = usePostsStore();
-    store.setPosts(tagSlug, categorySlug)
+    store.fetchPosts(tagSlug, categorySlug)
+    if (store.getTags.length == 0) {
```

Comment:
wenn du jetzt in einem anderen container ebenfalls die Tags benötigst, wirst du diese logik nochmals implementieren müssen. ich würde das direkt in die action im store verschieben.